pkg/controller/backup_controller_test.go

```diff
@@ -201,7 +203,14 @@ func TestProcessBackup(t *testing.T) {
 				backup.Status.Version = 1
 				backupper.On("Backup", backup, mock.Anything, mock.Anything, mock.Anything).Return(nil)
 
-				cloudBackups.On("UploadBackup", "bucket", backup.Name, mock.Anything, mock.Anything, mock.Anything).Return(nil)
+				// Ensure we don't have a nil CompletionTimestamp when uploading.
+				completionTimestampIsPresent := func(buf *bytes.Buffer) bool {
```

Comment:
we should be able to test for the specific value, right? Rather than just non-null?